Tasks/Common/ServiceFabricHelpers/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -7,5 +7,6 @@
   "loc.messages.ErrorOnAcquireToken": "An error occurred attempting to acquire an Azure Active Directory token. Ensure that your service endpoint is configured properly with valid credentials. Error message: {0}",
   "loc.messages.ErrorOnCertificateImport": "An error occurred attempting to import the certificate. Ensure that your service endpoint is configured properly with a correct certificate value and, if the certificate is password-protected, a valid password. Error message: {0}",
   "loc.messages.ImportedCertificate": "Imported cluster client certificate with thumbprint '{0}'.",
-  "loc.messages.ServiceFabricSDKNotInstalled": "Service Fabric SDK is not installed on the build agent.  Go to https://aka.ms/servicefabric to download and install it."
+  "loc.messages.ServiceFabricSDKNotInstalled": "Service Fabric SDK is not installed on the build agent.  Go to https://aka.ms/servicefabric to download and install it.",
+  "loc.messages.UseWindowsCredential": "Windows credential is specified. -WindowsCredential switch will be used while connecting."
```

Comment:
changed it to "UseWindowsSecurity". This is the term used by SF doc https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-windows-cluster-windows-security